crates/formality-rust/src/trait_binder.rs

```diff
@@ -25,6 +27,39 @@ where
 
 impl<T> CoreTerm<FormalityLang> for TraitBinder<T> where T: Term {}
 
+impl<T> Fuzzable<FormalityLang> for TraitBinder<T>
+where
+    T: CoreTerm<FormalityLang>,
+{
+    fn is_binder() -> bool {
+        true
+    }
+
+    fn estimate_cardinality(cx: &mut formality_core::fuzz::FuzzCx<'_, FormalityLang>) -> f64 {
+        cx.enter_estimate_cardinality::<Self>(|guard| {
+            (guard.binder_range().len() + 1) as f64 // N variables (including 1 extra for self)
+                * guard.estimate_cardinality::<ParameterKind>() // each can be either kind
```

Comment:
```suggestion
(guard.estimate_cardinality::<ParameterKind>().powi(guard.binder_range().len() as i32) + 1) 
```
we only have one parameter kind fort he self type, and again pow vs product